schemas/openshift/openshift-cluster-manager-1.yml

```diff
@@ -55,6 +76,9 @@ properties:
       required:
       - name
       - upgradePolicy
+dependencies:
+  upgradePolicyDefaults:
+  - upgradePolicyClusters
```

Comment:
I don't understand this section. What does that do ?

Reply:
it means that `upgradePolicyClusters` must exist if `upgradePolicyDefaults` is defined